lagtraj/domain/sources/era5/utils.py

```diff
@@ -11,12 +11,18 @@
 from .... import njit
 from .constants import rg, rd, rv_over_rd_minus_one
 
-levels_file = Path(__file__).parent / "137levels.dat"
-levels_table = pd.read_table(levels_file, sep=r"\s+")
-# skip the first row, as it corresponds to the top of the atmosphere
-# which is not in the data
-a_coeffs_137 = levels_table["a[Pa]"].values[1:]
```

Comment:
Is the zeroth entry still removed?

Reply:
Yes, I think that is handled by the `.sel(levels=...)` on https://github.com/EUREC4A-UK/lagtraj/pull/133/files#diff-54dd7f16e9f1835cf8a57cb819e1e512e9e0149029e5c04e42cfa6afc404b4bcR102. That ensures that only the levels needed are extracted and so will exclude the atmosphere top since we don't have that level in our data. Does that make sense?

Reply:
ok, let's chat about this on our next call 👍